lib/can-move-to-space.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+function canMoveToSpace(gameState, spaceIndex) {
+    const spaceInfo = {};
+    const isPlayerOne = gameState.isPlayerOne;
+    const boardSpace = gameState.board[spaceIndex];
+    
+    if (boardSpace.isPlayerOne === isPlayerOne) {
+        spaceInfo.isActivePlayer = true;
```

Comment:
If everything beyond this point in the function is wrapped in an `else`, then you could just `return` here directly.

```js
```